server/go/api_create_new_static_route_service.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Kusk Gateway API
+ *
+ * This is the Kusk Gateway Management API
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+import (
+	"context"
+	"net/http"
+	"errors"
+)
+
+// CreateNewStaticRouteApiService is a service that implements the logic for the CreateNewStaticRouteApiServicer
+// This service should implement the business logic for every endpoint for the CreateNewStaticRouteApi API.
+// Include any external packages or services that will be required by this service.
+type CreateNewStaticRouteApiService struct {
```

Comment:
Why not have a `StaticRouteApiService` that will handle creating, deleted etc rather than a service for each action?

Reply:
that's where we inject the interface and call respective methods for each operation see this example https://github.com/kubeshop/kuskgateway-api-server/blob/main/server/go/api_apis_service_impl.go#L27-L29